### PR TITLE
Translate toppage into japanese

### PR DESCRIPTION
**Pull request by nahokomatsui**

### CODE REVIEW

File: assets/translations/footer.js [hunk 1]
```diff
@@ -29,4 +29,10 @@ export default {
 		press: 'Press Kit',
 		charity: 'A 501(c)(3) Public Charity',
 	},
+	jp: {
+		credits: 'Credits',
+		contact: 'Contact Us',
+		press: 'Press Kit',
+		charity: 'A 501(c)(3) Public Charity',
+	},
```
Comment (nahokomatsui) [inline assets/translations/footer.js:37]: Leaving as English because I think English page header & footer are natural for Japanese website.

**Comment by jlooper** [comment]
I will take a look and push shortly, thank you🦊❤️🦊❤️

**Comment by jlooper** [comment]
hi, Nahoko, I pushed this PR's changes to our dev branch, which is built here: https://vue-vixens-dev.netlify.app/ - but as you can see the top nav is broken now. Would you mind taking a look, and seeing if you can fix that? PRs can go to the dev branch, thanks!

File: assets/translations/home.js [hunk 2]
```diff
@@ -114,4 +120,32 @@ export default {
...
+			content: 'Students and Mentors, learn more about our program.',
+		},
+		request: {
+			title: 'Request',
+			content: 'Conference organizers, register to host a workshop.',
+		},
+		homebox: {
+			workshopsAndEvents: 'ワークショップ＆イベント',
+			helpingWomen: '女性のメンターに教わろう！',
+			reactWorkshops: 'React ワークショップ 準備中'
```
Comment (ryamakuchi) [inline assets/translations/home.js:148]: 👍

File: assets/translations/home.js [hunk 3]
```diff
@@ -5,8 +5,9 @@ export default {
 			description:
 				'Front-End Foxes are people who identify as women and who want to learn front-end technologies' +
 				' to make websites and mobile apps',
-			workshops: 'Workshops',
-			patreon: 'Become a Patron',
+			workshops: 'Vue Workshops',
+			meetups: 'Meetups',
```
Comment (ryamakuchi) [inline assets/translations/home.js:9]: Is Meetups currently in use? 👀 
FYI: https://frontendfoxes.slack.com/archives/GC7FZPGMQ/p1588696755062500

Comment (nahokomatsui) [reply assets/translations/home.js:9]: @jlooper Is it still in use?
@ryamakuchi `Meetups` button is still present so I think I will leave the translation message on this PR though.

**Comment by nahokomatsui** [comment]
@ryamakuchi 
Thank you for your review.

Changes seem to be merged into dev branch. I will fix this on another PR :)
> The API URL path seems to be different.

File: assets/translations/home.js (hunk 3 above)
Comment (ryamakuchi) [reply assets/translations/home.js:9]: LGTM! 😸

**Pull request closed without merging**